### ISSUE TITLE
Dependencies don't resolve correctly when passed as an array

### ISSUE THREAD

**Issue by joostdecock** [freesewing/freesewing#971]
**What seems to be the problem?**

This problem was originally reported by Simon in Discord as such:

```
What I am doing:
1. The store value is set in part back.
2. I then access the store value from front

Different ways of declaring the dependency and whether it works or not:

dependencies: {front:['back']} - success
dependencies: {back:'front'}, - success

dependencies: {front:'back'} - fail
dependencies: {back:['front']} - fail

There seems to be an inversion of key/value on whether it is declared as a string or a list
```

**Additional context**

I wrote a unit test that falls over on this particular issue. Fixing the test should fix the issue :)

**Comment by joostdecock** [comment]
Tentatively fixed in https://github.com/freesewing/freesewing/commit/c7395cd9d9e7d7dfbb6c1533fd82fd2a18eda2e7

Keeping this open until we have an idea of the impact on existing patterns

**Comment by joostdecock** [comment]
This fix is available in 2.15.0-rc.0 which you can install with:

```
npm install @freesewing/core@next
```

**Comment by joostdecock** [comment]
Confirmed as fixed
